MAUI/Barcode/customization.md

```diff
@@ -0,0 +1,364 @@
+
+![.NET MAUI Barcode Generator Customization Show Text](images/customization/maui-customization-show-text.png)
+
+### Text style customization
+
+The style of the text can be customized with the `TextStyle` property of the barcode generator.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="6 7 8 9" %}
+
+            <barcode:SfBarcodeGenerator Value="12634388927" 
+                                        HeightRequest="150"
+                                        WidthRequest="300" 
+                                        ShowText="True">
+                <barcode:SfBarcodeGenerator.TextStyle>
+                    <barcode:BarcodeTextStyle FontAttributes="Italic" 
+                                              FontSize="16" 
+                                              FontFamily="Times" 
+                                              TextColor="Red"/>
+                </barcode:SfBarcodeGenerator.TextStyle>
+            </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="6 7 8 9 10 11 12" %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 300;
+        barcode.Value = "12634388927";
+        barcode.ShowText = true;
+        barcode.TextStyle = new BarcodeTextStyle()
+        {
+            FontAttributes = FontAttributes.Italic,
+            FontFamily = "Time",
+            FontSize = 16,
+            TextColor = Colors.Red
+        };
+
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Text Customization](images/customization/maui-text-customization.png)
+
+### Text spacing
+
+The space between the text and the barcode can be controlled by the `TextSpacing` property of barcode generator. By default, the value of `TextSpacing` is 2.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+            <barcode:SfBarcodeGenerator Value="12634388927" 
+                                        HeightRequest="150"
+                                        WidthRequest="300" 
+                                        ShowText="True" 
+                                        TextSpacing="25">
+            </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="6" %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 300;
+        barcode.Value = "12634388927";
+        barcode.ShowText = true;
+        barcode.TextSpacing = 25;
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Text Spacing](images/customization/maui-text-spacing.png)
+
+### Horizontal text alignment
+
+The horizontal alignment of the text can be controlled by `TextAlignment` property of barcode generator. The displayed value can be positioned at `Start`, `Center` or `End` of the control. The default value of `TextAlignment` property is center.
```

Comment:
The horizontal alignment of the text can be controlled by using the `TextAlignment` property of the barcode generator. The displayed value can be positioned at `Start`, `Center,` or `End` of the control.

Reply:
Changes updated.